package.json

```diff
@@ -5,15 +5,69 @@
     "type": "git",
     "url": "https://github.com/brownhci/WebGazer.git"
   },
-  "devDependencies": {
-    "git": "^0.1.5",
-    "grunt": "*",
-    "grunt-contrib-concat": "*",
-    "grunt-contrib-copy": "^1.0.0",
-    "grunt-contrib-uglify-es": "*",
-    "jsdoc": "^3.6.3"
+  "main": "dist/webgazer.commonjs2.js",
+  "scripts": {
+    "lint": "eslint .",
+    "clean": "rimraf dist",
+    "prebuild": "npm run clean",
+    "build": "webpack --progress --colors --config webpack.config.js --mode=production",
+    "prerelease": "npm run lint && npm test && npm run build"
+  },
+  "files": [
+    "dist",
+    "src",
+    "dist/webgazer.min.js"
+  ],
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm run build"
+    }
+  },
+  "lint-staged": {
+    "*.js": [
+      "eslint --fix",
+      "git add"
+    ]
   },
+  "keywords": [
+    "webgazer"
+  ],
   "dependencies": {
-    "mime": "^2.4.3"
+    "@tensorflow-models/facemesh": "0.0.3",
+    "@tensorflow/tfjs-backend-webgl": "2.0.1",
```

Comment:
We should include the backends for WASM and CPU too as fallbacks. Once SIMD is enabled for WASM builds, it might even run faster.

Reply:
No sure how to set this up, maybe we should expose tf configs in config parameters. Fallback to cpu is automatic and thats already included in '@tensorflow/tfjs-core'

Reply:
According to the [changelog](https://github.com/tensorflow/tfjs/releases/tag/tfjs-v2.0.0) for tfjs 2.0 unless you are using the union package, which we are not, then you need to include the cpu-backend (tf-backend-cpu).

Reply:
Including the union package instead!